src/tests/suite/daos_degraded.c

```diff
@@ -1,16 +1,14 @@
 /**
- * (C) Copyright 2016-2022 Intel Corporation.
+ * (C) Copyright 2016-2023 Intel Corporation.
  *
  * SPDX-License-Identifier: BSD-2-Clause-Patent
  */
 /**
  * This file is part of daos
  *
- * tests/suite/daos_replicated.c
- *
  * Replication tests need external interaction, to
```

Comment:
Degraded tests?

Reply:
Could be.  I removed the filename because it was obviously wrong but don't want to get into correcting a comment.